keel-core/src/main/kotlin/com/netflix/spinnaker/keel/model/Capacity.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.model
+
```

Comment:
This is a general spinnaker class, so putting it here so it's not duplicated in every plugin.

Reply:
Quibble with package naming. I think everything that is part of the external facing API should be in `com.netflix.spinnaker.keel.api`.

Reply:
ok, i'll move